pkg/proxy/config/config.go

```diff
@@ -54,7 +54,7 @@ type EndpointsHandler interface {
 	// OnEndpointsUpdate is called whenever modification of an existing
 	// endpoints object is observed.
 	OnEndpointsUpdate(oldEndpoints, endpoints *v1.Endpoints)
-	// OnEndpointsDelete is called whever deletion of an existing endpoints
+	// OnEndpointsDelete is called whenever deletion of an existing endpoints
```

Comment:
There is a same typo at line 559 of `pkg/proxy/iptables/proxier.go`.

Reply:
got it, let me fix

Reply:
@atoato88 just fixed, sry for delay. Could you help to review again?

Reply:
@haosdent Thank you for fixing typo in `pkg/proxy/iptables/proxier.go`.
It's ok for me at point I commented.
Please follow comments from other member.

Reply:
Got it, thx @atoato88 's help Orz